.github/workflows/push-images.yml

```diff
@@ -73,7 +73,8 @@ jobs:
         run: |
           set -x
 
-          for tag in ${{ join(steps.modelmeta.outputs.tags, ' ') }}; do
```

Comment:
`steps.modelmeta.outputs.tags` is not an array, but a string delimited with a newline.